### ISSUE TITLE
Creating a template to report a bug as a new issue

### ISSUE THREAD

**Issue by mkruckow** [POSYDON-code/POSYDON-public#415]
Github has the possibility to create templates for new issues.
It would be useful to have at least a template for reporting bugs on pop-syn. It should guide the person finding the issue to provide all needed information, e.g. with parameters have been used, which gird version, ... In best a [MWE](https://stackoverflow.com/help/minimal-reproducible-example) should be provided which can be used to quickly (hence only evolving a single binary) reproduce the issue.
It turned out, that I don't have the permissions to set this up.

**Comment by tassos25** [comment]
I agree :) But I guess you tagged me expecting something more than approval?

**Comment by mkruckow** [comment]
> But I guess you tagged me expecting something more than approval?

Yes, you got tagged, because I expect you to have the permission to either implement it yourself and/or give the needed permission to someone whom you think should implement it. In the latter case you should reassign the issue to the person you gave permissions to.
